sdk/core/core-http/test/resources/httpbin-index.html

```diff
@@ -221,7 +221,7 @@ <h2 id="SEE-ALSO">SEE ALSO</h2>
     window._pa = window._pa || {};
     _pa.productId = "httpbin";
     var pa = document.createElement('script'); pa.type = 'text/javascript'; pa.async = true;
-    pa.src = ('https:' == document.location.protocol ? 'https:' : 'http:') + "//tag.perfectaudience.com/serve/5226171f87bc6890da0000a0.js";
+    pa.src = ('https:' === document.location.protocol ? 'https:' : 'http:') + "//tag.perfectaudience.com/serve/5226171f87bc6890da0000a0.js";
```

Comment:
hopefully this doesn't break any tests?

Reply:
since it's a test resource and the length changed, I could see that causing failures somewhere

Reply:
Oh core-http tests pass. ai-form-recognizer tests don't pass after these changes though. I am trying to figure out why.

Reply:
It was an automated eqeqeq fix on src/serializer.ts . It doesn't seem trivial to fix this. That whole file and other files require of considerable cleanup to remove the ambiguities.

Reply:
What's also interesting is that none of the core-http tests broke this, but the ai-form-recognizer tests did catch the bug.